### PR TITLE
[reggen/doc] Fix a incorrect register references to multiregs

### PR DESCRIPTION
**Pull request by msfschaffner**
See #9237 for reference.

### CODE REVIEW

File: util/reggen/html_helpers.py [hunk 1]
```diff
@@ -40,7 +40,16 @@ def _expand_paragraph(s: str, rnames: Set[str]) -> str:
     '''Expand a single paragraph, as described in _get_desc_paras'''
     def fieldsub(match: Match[str]) -> str:
         base = match.group(1).partition('.')[0].lower()
-        if base in rnames:
+
+        # If we do not find the register name, there is a chance that this
```
Comment (tjaychen) [inline util/reggen/html_helpers.py:44]: o nice!

**Comment by msfschaffner** [comment]
Accidentally closed this, sorry for the spam here.
Will come back to this at a later point.

**Comment by msfschaffner** [comment]
I finally had some time to come back to this.
I combined solution 2. with @rswarbrick's suggestion in order to avoid large diffs in RTL, DV and DIF code.

**Review by vogelpi — APPROVED** [review]
This looks all great, thanks @msfschaffner !

**Pull request merged**